packages/grid_client/src/clients/rmb/client.ts

```diff
@@ -12,7 +14,14 @@ class RMB {
       const requestId = await this.client.send(cmd, payload, destTwinIds[0], expiration / 60);
       result = await this.client.read(requestId);
     } catch (e) {
-      throw Error(
+      if (e instanceof BaseError) {
+        e.message = formatErrorMessage(
+          `Failed to send request to twinId ${destTwinIds} with command: ${cmd}, payload: ${payload}`,
+          e,
```

Comment:
Same error message is being used in the catch twice. Can we store it in a variable and use it instead?